docs/source/checks/tabular/model_evaluation/plot_model_inference_time.py

```diff
@@ -26,13 +41,15 @@
 clf = AdaBoostClassifier().fit(train_ds.data[train_ds.features], train_ds.data[train_ds.label_name])
 
 #%%
-
+# Run the check
+# =============
 check = ModelInferenceTime()
 check.run(test_ds, clf)
 
 #%%
-# Instantiating check instance with condition
-# ===========================================
-
+# Define a condition
+# ==================
+# A condition for the average model inference time can also be passed as a parameter.
```

Comment:
What do you mean here, by "passed as a parameter?"

Reply:
`check = ModelInferenceTime().add_condition_inference_time_less_than(0.00001)`

0.00001 is the parameter to the function.

maybe use "variable"?

Reply:
Hmm. I understand what you're trying to say, but to me it sounds like the condition is passed as a parameter. Moreover, the "also" there confuses me, as I don't see what it "continues"